doc/using/configuration.chapter.md

```diff
@@ -164,6 +166,42 @@ There are several ways to tweak how Nix handles a package which has been marked
 
     Note that `permittedInsecurePackages` is only checked if `allowInsecurePredicate` is not specified.
 
+## Installing packages with warnings {#sec-ignore-warnings}
+
+There are several ways to tweak how Nix handles a package which has some warnings.
+
+-   To temporarily allow ignore all warnings, you can use an environment variable for a single invocation of the nix tools:
+
+    ```ShellSession
+    $ export NIXPKGS_IGNORE_WARNINGS=1
+    ```
```

Comment:
`export`ing a variable holds for the shell session. Either make a different example such as


```suggestion
-   To temporarily ignore all warnings, you can use an environment variable for a single invocation of Nix commands:

    ```ShellSession
    $ env NIXPKGS_IGNORE_WARNINGS=1 nix-shell -p hello
    ```
```

or reword

```suggestion
-   To temporarily ignore all warnings, you can use an environment variable for modifying Nix configuration:

    ```ShellSession
    $ export NIXPKGS_IGNORE_WARNINGS=1
    ```
```

Reply:
I am with you, however I mostly copied the wording over from `pkgs/stdenv/generic/check-meta.nix` messages and changed no more than necessary. So I suggest fixing all of them at once, to remain consistent

Reply:
In turn, I am with you that you probably don't want to touch unrelated parts, but there is a dilemma: proliferating errors (or confusion, as in this case) in documentation creates more work (and therefore a higher barrier to contributing) for whoever wants to fix this. I know this kind of review comment is super annoying because it tries to pry open your PR's scope. In this case I didn't know, but would still kindly ask you to consider. In my experience, as the manual is so massive, there is usually just one rare chance to iron out kinks like these: drive-by style, while there is some attention to the issue.